x-pack/plugins/security_solution/public/network/components/embeddables/__mocks__/mock.ts

```diff
@@ -7,6 +7,7 @@
 
 import { IndexPatternMapping } from '../types';
 import { IndexPatternSavedObject } from '../../../../common/hooks/types';
+import { LAYER_TYPE } from '../../../../../maps/common';
```

Comment:
this just needs one more `../`:
import { LAYER_TYPE } from '../../../../../../maps/common';